src/main/java/jp/co/soramitsu/sora/crypto/service/impl/JSONSignerImpl.java

```diff
@@ -0,0 +1,59 @@
+package jp.co.soramitsu.sora.crypto.service.impl;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.io.IOException;
+import java.security.Signature;
+import java.security.SignatureException;
+import jp.co.soramitsu.sora.common.Util;
+import jp.co.soramitsu.sora.crypto.common.Consts;
+import jp.co.soramitsu.sora.crypto.proof.Options;
+import jp.co.soramitsu.sora.crypto.proof.Proof;
+import jp.co.soramitsu.sora.crypto.service.JSONCanonizer;
+import jp.co.soramitsu.sora.crypto.service.JSONSigner;
+import lombok.AllArgsConstructor;
+import lombok.NonNull;
+
+
+@AllArgsConstructor
+public class JSONSignerImpl implements JSONSigner {
```

Comment:
add annotation @FieldDefaults, in this case you could to remove private and final keywords